include/boost/url/impl/any_allocator.hpp

```diff
@@ -0,0 +1,227 @@
+//
+// Copyright (c) 2022 Alan Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CPPAlliance/url
+//
+
+#ifndef BOOST_URL_IMPL_ANY_ALLOCATOR_HPP
+#define BOOST_URL_IMPL_ANY_ALLOCATOR_HPP
+
+#include <atomic>
+
+#include <boost/url/detail/over_allocator.hpp>
+#include <boost/type_traits/make_void.hpp>
+#include <boost/type_traits/enable_if.hpp>
```

Comment:
I'd prefer `std::enable_if`

Reply:
Oh... OK. I tend to assume you almost always prefer boost. :)